protobuf/coreapi/api.proto

```diff
@@ -42,6 +42,9 @@ service Core {
 
   // ServiceLogs gives a stream for dependency logs of a service.
   rpc ServiceLogs (ServiceLogsRequest) returns (stream LogData) {}
+
+  // Info returns all necessary informations from the core.
```

Comment:
Better comment (all necessary info but for what?)

s/Informations/Information/